protocol/test-serialization/src/test_3NKaBJsN1SehD6iJwRwJSFmVzJg5DXSUQVgnMxtH4eer4aF5BrDK.rs

```diff
@@ -59,12 +192,39 @@ mod tests {
             "867667132.840039233"
         );
 
+        let constants = &body.constants;
+        assert_eq!(constants.k.0, 290);
+        assert_eq!(constants.slots_per_epoch.0, 7140);
+        assert_eq!(constants.slots_per_sub_window.0, 7);
+        assert_eq!(constants.delta.0, 0);
+        assert_eq!(
+            constants.genesis_state_timestamp.epoch_millis(),
+            1615939200000
+        );
+
+        // TODO: Verify protocol_state_proof
```

Comment:
What's the blocker for this?

Reply:
Willem is working on refactoring this part, we'd better do this later